src/main/resources/static/widget.js

```diff
@@ -0,0 +1,36 @@
+function getIframe() {
+  let tcTokenURL = new URL(document.currentScript.src).searchParams.get(
+    "tcTokenURL"
+  );
+  if (tcTokenURL === null) {
+    tcTokenURL = document.getElementById("useid-widget").dataset.tcTokenUrl;
+
+    if (tcTokenURL === undefined) {
+      const error = document.createElement("div");
+      error.innerHTML = "Fehlerhafte Konfiguration: TC Token nicht definiert.";
+      return error;
+    }
+  }
+
+  const iframe = document.createElement("iframe");
+  iframe.setAttribute(
+    "src",
+    `https://useid.dev.ds4g.net/widget?tcTokenURL=${tcTokenURL}`
+  );
+  iframe.style.border = "2px solid #B8BDC3";
+  iframe.style.borderRadius = "20px";
+  iframe.style.width = "100%";
+  iframe.style.minHeight = "600px"; // TODO: Set height to fit
+  return iframe;
+}
+
+const container = document.getElementById("useid-widget-container");
+container === null
+  ? document.write(getIframe().outerHTML)
```

Comment:
We could insert a default container here with some styling (see comment above)

Reply:
So the parent page could still style it, even if they didn't implement it themselves? Yes, we can do that.